javav2/usecases/creating_photo_analyzer_app/src/main/java/com/example/photo/PhotoController.java

```diff
@@ -63,7 +69,7 @@ String report(HttpServletRequest request, HttpServletResponse response) {
        List myKeys =  s3Client.ListBucketObjects("scottphoto");
 
        // Create a List to store the data
-       List myList = new ArrayList<List>();
+       List<List> myList = new ArrayList<List>();
 
        // loop through each element in the List
```

Comment:
loop -> Loop and a period at the end of the sentence.

Reply:
fixed